src/main/java/command/FindCommand.java

```diff
@@ -0,0 +1,59 @@
+package command;
+
+import exception.FindException;
+import storage.Storage;
+import task.Task;
+import task.TaskList;
+import ui.Ui;
+
+import java.util.ArrayList;
+
+/**
+ * Command for finding query in TaskList.
+ */
+public class FindCommand extends Command {
+
+    protected String[] input;
+
+    public FindCommand(String[] input) {
+        this.input = input;
+    }
+
+    /**
+     * execute command for finding query in user's tasks list.
+     * @param tasks a TaskList.
+     * @param ui User Interface for output printing.
+     * @param storage hard disk storage.
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        if (input.length <= 1) {
+            System.err.println("     " + new FindException());

```

Comment:
Hmm would it be better if you handled your exceptions in one place rather than just printing the exception itself in every command?